### ISSUE TITLE
break statement

### ISSUE THREAD

**Issue by thuld** [seanmcne/Microsoft.Xrm.Data.PowerShell#211]
In line [5041 ](https://github.com/seanmcne/Microsoft.Xrm.Data.PowerShell/blob/master/Microsoft.Xrm.Data.PowerShell/Microsoft.Xrm.Data.PowerShell.psm1#L5041)are you using an break statement:

````
Write-Warning 'You need to create Connect to CRM Organization. Use Get-CrmConnection to create it.'
break
````

Due to this break is my script stopped and there is nothing from stopping PowerShell from closing my PowerShell window.

**Comment by seanmcne** [comment]
interesting, I'm not entirely sure why a break statement was used there.  I think we should probably flip it over to a throw instead and blow back an error instead.  I suspect it wasn't discovered before as it wasn't run within a loop maybe?  

The error is valid though, was this just causing some debug issues when $conn wasn't initialized and a connection wasn't passed in?

**Comment by seanmcne** [comment]
Fixed in #212
